### ISSUE TITLE
Add a contributors file

### ISSUE THREAD

**Issue by matthewfeickert** [dguest/pandamonium#55]
`pandamonium` is made better by the people who were kind enough to hack on it and make a drive-by PR. Giving them a nod by adding their name to a `contributors.md` file would be nice.

This is motivated by how [`pyhf` does things](https://github.com/scikit-hep/pyhf/blob/4b859578b4eb86c9167004f9909c8dc79ff41a41/docs/contributors.rst) but you can get [even fancier like `awkward` does with a bot](https://github.com/scikit-hep/awkward-1.0#acknowledgements).

**Comment by dguest** [comment]
isn't that what the zenodo file does?

**Comment by matthewfeickert** [comment]
No, the Zenodo config file is meant to be for the authors / copyright holders.

https://github.com/dguest/pandamonium/blob/5650c0de68f74644489984ae576041f319e4686b/.zenodo.json#L7-L18

but if you don't tell Zenodo anything then it thinks that everyone who has contributed is an author / owner.
